s3_upload/utils/utils.py

```diff
@@ -327,8 +327,19 @@ def get_sequencing_file_list(seq_dir, exclude_patterns=None) -> list:
     -------
     list
         sorted list of files by their file size (descending)
+
+    Raises
+    ------
+    FileNotFoundError
+        Raised if the provided directory does not exist / not accessible
     """
     log.info("Getting list of files to upload in %s", seq_dir)
+
+    if not Path(seq_dir).absolute().exists():
+        raise FileNotFoundError(
+            f"Provided directory does not exist: {seq_dir}"
+        )
```

Comment:
added this to the log, alerts not needed since this will only be valid for the interactive single upload mode. When run in monitor mode the provided monitor dirs are provided and check at the beginning that they are valid and alerted from there